run_all_unit_tests.sh

```diff
@@ -15,6 +15,7 @@ lockfailed()
 # create an environment including the set of requirements specified in fuse-med-ml/requirements.txt (if not already exist)
 create_env() {
     force_cuda_version=$1
+    dest_path=$2
```

Comment:
maybe I misunderstand, but isn't there a confusion here between "dest_path" and "env_path" ?